src/main/java/io/confluent/connect/hdfs/orc/OrcHiveFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.connect.hdfs.orc;
+
+import io.confluent.connect.hdfs.HdfsSinkConnectorConfig;
+import io.confluent.connect.hdfs.hive.HiveMetaStore;
+import io.confluent.connect.storage.hive.HiveFactory;
+import io.confluent.connect.storage.hive.HiveUtil;
+import org.apache.kafka.common.config.AbstractConfig;
+
+public class OrcHiveFactory implements HiveFactory {
+
+  @Override
+  public HiveUtil createHiveUtil(
+      AbstractConfig conf,
+      io.confluent.connect.storage.hive.HiveMetaStore hiveMetaStore
+  ) {
+    return createHiveUtil((HdfsSinkConnectorConfig) conf, (HiveMetaStore) hiveMetaStore);
+  }
+
+  @Deprecated
```

Comment:
I haven't looked at the details too far, but I believe it'd be safe to skip the deprecated interfaces/methods. 

This format is introduced after the transition to storage-common, and therefore there's no obvious reason to retain compatibility with the deprecated stuff

Reply:
fixed